README.md

```diff
@@ -53,6 +53,14 @@ module.exports = function (fastify, opts, next) {
 // optional
 module.exports.autoload = false
 ```
+
+If you want to pass some custom options to the registered plugins via `fastify-autoload`, use the `options` key:
+```js
+fastify.register(AutoLoad, {
+  dir: path.join(__dirname, 'foo'),
+  options: { foo: 'bar' }
+})
+```
```

Comment:
are the options shared by all plugins? Are they prefixed?

Reply:
Yes, it is passed to all the auto registered plugins.
If user want different options based on the plugin it can prefix the options.
```
fastify.register(AutoLoad, {
  dir: path.join(__dirname, 'foo'),
  options: {
    auth: {},
    endpoint: {},
  }
})
```

Since internally we are using `Object.assign` the prefix property it is not lost.

Reply:
Can you make this explicit? "options will be passed to all loaded plugins".

Reply:
Done :)